plugins/hydra_ax_sweeper/tests/test_ax_sweeper_plugin.py

```diff
@@ -159,6 +157,9 @@ def test_jobs_configured_via_cmd_and_config(hydra_sweep_runner: TSweepRunner) ->
         assert math.isclose(best_parameters["quadratic_x"], -2.0, abs_tol=1e-4)
         assert math.isclose(best_parameters["quadratic_y"], 1.0, abs_tol=1e-4)
 
+        assert math.isclose(best_parameters["quadratic_x"], -2, abs_tol=1e-4)
+        assert math.isclose(best_parameters["quadratic_y"], 1, abs_tol=1e-4)
```

Comment:
Since this is a Hydra plugin, I am assuming best_parameters is a DictConfig.
use attribute style access.